go-ussd_registration.js

```diff
@@ -1377,8 +1378,34 @@ go.app = function() {
                     "Please dial back into {{channel}} to complete the Hello MAMA registration"
                     );
             });
-        };
 
+            self.env = self.im.config.env;
+            self.metric_prefix = [self.env, self.im.config.name].join('.').replace(/-/g, '_');
+            self.store_name = [self.env, self.im.config.name].join('.');
+
+            mh = new MetricsHelper(self.im);
+            mh
+                .add.trigger({
+                    action: 'enter',
+                    state: 'state_msg_receiver'
+                }, {
+                    total_state_actions: [self.metric_prefix, "registrations_started"].join('.'),
+                })
+                .add.trigger({
+                    action: 'enter',
+                    state: 'state_end_voice'
+                }, {
+                    total_state_actions: [self.metric_prefix, "registrations_completed"].join('.'),
+                })
+                .add.trigger({
+                    action: 'enter',
+                    state: 'state_end_sms'
+                }, {
+                    total_state_actions: [self.metric_prefix, "registrations_completed"].join('.'),
+                })
```

Comment:
Sorry, one more thing, do we have plans to add more metrics here in the future? Maybe it might be a little cleaner to use the `total_state_actions` trigger directly: http://go-jsbox-metrics-helper.readthedocs.io/en/latest/metricshelper.html#MetricsHelper.add.total_state_actions